argoverse/data_loading/argoverse_stereo_loader.py

```diff
@@ -0,0 +1,440 @@
+
+        assert self.calib is not None
+
+    @property
+    def calib(self) -> Dict[str, Calibration]:
+        """get calibration dict for current log
+
+        Returns:
+            calib: Calibration object for the current log
+        """
+        self.__ensure_calib_is_populated()
+        assert self._calib is not None
+        return self._calib[self.current_log]
+
+    def __ensure_calib_is_populated(self) -> None:
+        """load up calibration object for all logs
+
+        Returns:
+            None
+        """
+        if self._calib is None:
+            self._calib = {}
+            for log in self.log_list:
+                calib_filename = os.path.join(
+                    self.root_dir,
+                    "rectified_stereo_images_v1.1",
+                    self.split_name,
+                    log,
+                    "vehicle_calibration_stereo_info.json",
+                )
+                self._calib[log] = load_stereo_calib(calib_filename)
+
+    @property
+    def log_list(self) -> List[str]:
+        """return list of log (str) in the current dataset directory
+
+        Returns:
+            log_list: list of string representing log id
+        """
+        if self._log_list is not None:
+            return self._log_list
+
+        def valid_log(log: str) -> bool:
+            return os.path.exists(
+                os.path.join(
+                    self.root_dir,
+                    "rectified_stereo_images_v1.1",
+                    self.split_name,
+                    log,
+                    "vehicle_calibration_stereo_info.json",
+                )
+            )
+
+        self._log_list = [
+            x
+            for x in os.listdir(os.path.join(self.root_dir, "rectified_stereo_images_v1.1", self.split_name))
+            if valid_log(x)
+        ]
+
+        return self._log_list
+
+    @property
+    def image_list(self) -> Dict[str, List[str]]:
+        """return list of all image path (str) for the current log
+
+        Returns:
+            image_list: dictionary of list of image, with camera name as key
+        """
+        if self._image_list is None:
+            self._image_list = {}
+            for log in self.log_list:
+                self._image_list[log] = {}
+                for camera in RECTIFIED_STEREO_CAMERA_LIST:
+                    self._image_list[log][camera] = sorted(
+                        glob.glob(
+                            (
+                                os.path.join(
+                                    self.root_dir, "rectified_stereo_images_v1.1", self.split_name, log, camera, "*.jpg"
+                                )
+                            )
+                        )
+                    )
+                    self.image_count += len(self._image_list[log][camera])
+        return self._image_list[self.current_log]
+
+    @property
+    def disparity_list(self) -> Dict[str, List[str]]:
```

Comment:
nit: capitalize Return

Reply:
yeah, just fixed all docstrings that were initialized with low caps.